src/routes/formsgSiteLaunch.ts

```diff
@@ -162,16 +169,26 @@ export class FormsgSiteLaunchRouter {
     // 1. Extract arguments
     const { submissionId } = req.body.data
     const { responses } = res.locals.submission
-    const formResponses: FormResponsesProps = {
-      submissionId,
-      requesterEmail: getField(responses, REQUESTER_EMAIL_FIELD),
-      repoName: getField(responses, REPO_NAME_FIELD),
-      primaryDomain: getField(responses, PRIMARY_DOMAIN),
-      redirectionDomain: getField(responses, REDIRECTION_DOMAIN),
-      agencyEmail: getField(responses, AGENCY_EMAIL_FIELD),
-    }
+    const siteLaunchList = getFieldsFromTable(responses, SITE_LAUNCH_LIST)
+    const formResponses: FormResponsesProps[] = []
+    siteLaunchList?.forEach((element) => {
```

Comment:
nit: could we do this as a `map` operation as this doesn't seem to have a side effect

Reply:
4b9cc76